pages/index.js

```diff
@@ -1,9 +1,79 @@
-import Feed from "./components/feed";
+import Head from "next/head";
+import Image from "next/image";
+import Link from "next/link";
+import { Inter } from "next/font/google";
+import styles from "@/styles/Home.module.css";
+import styled, { createGlobalStyle } from "styled-components";
 
-export default function HomePage() {
+const inter = Inter({ subsets: ["latin"] });
+
+export default function Home() {
   return (
     <>
-      <Feed />
+      <Head>
+        <title>Music</title>
+        <meta name="description" content="Generated by create next app" />
+        <meta name="viewport" content="width=device-width, initial-scale=1" />
+        <link rel="icon" href="/favicon.ico" />
+      </Head>
+      <GlobalStyle />
+      <StyledBody>
+        <main className={`${styles.main} ${inter.className}`}>
+          <Title>
+            <h1>Share youre lovely music with Music</h1>
```

Comment:
I think you have to rephrase this. I am not sure what you mean with this sentence :)

Reply:
it is a dummy sentence. i will change it in the course of time